Jenkinsfile

```diff
@@ -0,0 +1,24 @@
+node {
+    stage ('Checkout') {
+        checkout scm
+    }
+    try {
+		stage ('Quality checks') {
+			parallel 'format': {
+				sh 'make format'
+			}, 'clippy': {
+				sh 'make lint'
+			}
+		}
+        stage ('Tests') {
+            sh 'make test'
+        }
```

Comment:
```suggestion
        stage ('Quality checks') {
	    parallel 'format': {
	        sh 'make format'
	    }, 'clippy': {
	        sh 'make lint'
	    }
        }
        stage ('Tests') {
            sh 'make test'
        }
```
indentation

Reply:
Still some mix of tab and spaces

Reply:
Yep, see below my "problem" of not pushing the update. Should be fixed now.